### PR TITLE
Make 'question description' text box taller and add a placeholder text #5871

### LINKED ISSUE #5871: Make 'question description' text box taller and add a placeholder text
It's better if the 'description' text box is resizeable just like the text box above it.
image

### PR DESCRIPTION
**Pull request by belyabl9**
Fixes #5871

### CODE REVIEW

**Comment by JosephineKwa** [comment]
Hey @belyabl9 , the question description box for new questions still seems to be rather short, as shown below:
qn

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionEditForm.tag [hunk 1]
```diff
@@ -90,7 +91,8 @@
                     </h5>
                     <div class="col-sm-10">
                         <div id="rich-text-toolbar-q-descr-container-${fqForm.questionIndex}"></div>
-                        <div class="well panel panel-default panel-body" rows="5"
+                        <div class="well panel panel-default panel-body" style="min-height:120px;"
```
Comment (JosephineKwa) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionEditForm.tag]: Could the styling be moved to a css file instead?

File: src/main/webapp/stylesheets/teammates.css [hunk 2]
```diff
@@ -378,4 +378,4 @@ h2.subcaption {
     margin: 2px 20px 0px 10px;
     width: 150px;
     text-align: center;
-}
+}
```
Comment (JosephineKwa) [inline src/main/webapp/stylesheets/teammates.css]: This change doesn't seem necessary

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 3]
```diff
@@ -947,3 +947,16 @@ Prevent Long URL From Breaking Out of Container
...
+}
+
+.content-editor:before {
+    display: none;
+}
+.content-editor.empty:before {
+    display: block;
+    color: gray;
+    content: attr(data-placeholder);
+}
```
Comment (JosephineKwa) [inline src/main/webapp/stylesheets/teammatesCommon.css]: Do add a newline here

**Comment by belyabl9** [comment]
Ready to review

**Pull request merged**